go/vt/vtgate/planbuilder/show.go

```diff
@@ -275,6 +275,23 @@ func buildPlanWithDB(show *sqlparser.ShowBasic, vschema ContextVSchema) (engine.
 
 }
 
+func buildShowStatusGlobalPlan(show *sqlparser.ShowBasic, vschema ContextVSchema) (engine.Primitive, error) {
```

Comment:
nit: can be renamed to `buildShowStatusPlan` as it is building plan for global and session status.